diff --git a/tests/test_sqs/test_sqs_cloudformation.py b/tests/test_sqs/test_sqs_cloudformation.py
--- a/tests/test_sqs/test_sqs_cloudformation.py
+++ b/tests/test_sqs/test_sqs_cloudformation.py
@@ -15,6 +15,6 @@ simple_queue = Template(
         "QueueGroup": {
             "Type": "AWS::SQS::Queue",
-            "Properties": {"QueueName": $q_name, "VisibilityTimeout": 60},
+            "Properties": {"QueueName": "$q_name", "VisibilityTimeout": 60},
         }
     },
 }"""
